packages/core/scripts/ensureGitignore.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
```

Comment:
Is this used anymore? We do need this as an internal crackle utility (because crackle apps won't commit the entrypoint package.jsons), so it should probably become part of core at some point.

If we're keeping it (as a standalone script), is it possible to have it typed?

Reply:
This script updates the `.gitignore` for `packages/core`. It can go because of [this](https://github.com/seek-oss/crackle/pull/14#discussion_r949779955) and [this](https://github.com/seek-oss/crackle/pull/14#discussion_r949780367).